src/database/models/ScheduledEventModel.ts

```diff
@@ -0,0 +1,13 @@
+import { model, Schema } from "mongoose";
+
+import { ScheduledEvent } from "../../interfaces/database/ScheduledEvent";
+
+export const ScheduledEventSchema = new Schema({
+  member: String,
+  time: Number,
+  targetChannel: String,
+  lang: String,
+  message: String,
+});
```

Comment:
```suggestion
export const ScheduledEventSchema = new Schema<ScheduledEvent>({
  member: String,
  time: Number,
  targetChannel: String,
  lang: String,
  message: String,
});
```

TypeScript seems to play better with these if we pass a generic.